### PR TITLE
[FSDE-12537] Configure Brightspeed theme

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSDE-12537

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-302/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://643f6a84887db926eda84108e4c5102f0aa5ba39-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-302-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-302/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b51f6a8709571d79b0a24995e9555ff6e58f77d7-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-302-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-302/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://47b6eef93ba6305a6f9fa04df5d3f3984808a820-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-302-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-302/12/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://90147117d9dc95d0d217d6c767c67f2e04cbbc4b-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-302-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/components/ThemeSwitcher.vue [hunk 1]
```diff
@@ -73,6 +73,21 @@
           src="/themes/lumen/images/favicon.svg"
         />
         <div class="-theme-name">Lumen Enterprise Portal</div></a
+      ><a
```
Comment (janibolkvadze) [inline src/documentation/components/ThemeSwitcher.vue:76]: @jllr Not related to the scope of this PR, but we will need to generate these elements from an array (in a separate PR) as the code is repeated for 4 themes now.

**Review by dani-cl-madrid — APPROVED** [review]

**Review by dani-cl-madrid — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by janibolkvadze** [comment]
@mattnickles Looks awesome!

**Comment by jllr** [comment]
@mattnickles great job!

**Pull request merged**